packages/owl-bot/cloud-build/scan-googleapis-gen-and-create-pull-requests.yaml

```diff
@@ -33,6 +33,12 @@ steps:
       - --global
       - user.name
       - 'Owl Bot'
+  - name: 'gcr.io/cloud-builders/git'
```

Comment:
My guess is we need to set $HOME for correctly preserving the configs.

(Also applies to the steps above)
@SurferJeffAtGoogle

Reply:
It looks to me like HOME is being set on lines 39 and 47.